fili-core/src/main/resources/moduleConfig.properties

```diff
@@ -124,6 +124,9 @@ bard__default_asyncAfter=never
 # Flag to turn on case sensitive keys in keyvalue store
 bard__case_sensitive_keys_enabled = false
 
+# Sets the upper limit of the number of Druid uncovered interval. Default is -1 meaning no uncovered interval is allowed
+bard__druid_uncovered_interval_limit = -1
```

Comment:
@garyluoex @michael-mclawhorn I have changed it to 0. https://github.com/yahoo/fili/pull/267 has been update as well.